csrc/ir_nodes.cpp

```diff
@@ -1447,8 +1447,8 @@ std::string TransposeOp::toInlineString(int indent_size) const {
 std::vector<int64_t> TransposeOp::old2new() const {
   std::vector<int64_t> old2new(new2old().size());
   for (auto new_axis : c10::irange(new2old().size())) {
-    auto old_axis = new2old().at(new_axis);
-    old2new[old_axis] = new_axis;
+    int old_axis = (int)new2old().at(new_axis);
```

Comment:
`new2old()` returns a vector of `int64_t`, so this cast to `int` shouldn't be necessary.